envs/monkey_zoo/terraform/firewalls.tf

```diff
@@ -156,6 +156,36 @@ resource "google_compute_firewall" "powershell-48-allow" {
   target_tags = ["powershell-48"]
 }
 
+
+resource "google_compute_firewall" "powershell-48-allow-egress-45" {
+  name    = "${local.resource_prefix}powershell-48-allow-egress-45"
+  network = google_compute_network.monkeyzoo.name
+
+  allow {
+    protocol = "all"
+  }
+
+  direction = "EGRESS"
+  priority = "5"
+  destination_ranges = ["10.2.3.45/32"]
+  target_tags = ["powershell-48"]
+}
+
+resource "google_compute_firewall" "powershell-48-deny-all-egress" {
+  name    = "${local.resource_prefix}powershell-48-deny-all-egress"
+  network = google_compute_network.monkeyzoo.name
+
+  deny {
+    protocol = "all"
+  }
+
+  direction = "EGRESS"
+  priority = "10"
+  destination_ranges = ["10.2.3.45/32"]
+  target_tags = ["powershell-48"]
```

Comment:
So this denies only traffic to the .45? how did you test?

Reply:
This should allow. I will fix.

Reply:
This should deny all traffic. Fixed it.